test/mri/tests/runner.rb

```diff
@@ -20,9 +20,6 @@ class Gem::TestCase < MiniTest::Unit::TestCase
 
 ENV["GEM_SKIP"] = ENV["GEM_HOME"] = ENV["GEM_PATH"] = "".freeze
 
-# Use MRI backtraces for tests that verify error output
-ENV["TRUFFLERUBYOPT"] = ENV["TRUFFLERUBYOPT"].to_s + " -Xbacktrace.style=mri"
```

Comment:
I wonder, how did this ever worked since we removed the option? Was it ignored before?

Reply:
I think we just did not check invalid `-X...` options.